src/components/views/settings/tabs/user/GeneralUserSettingsTab.tsx

```diff
@@ -427,7 +427,7 @@ export default class GeneralUserSettingsTab extends React.Component<IProps, ISta
             <div className="mx_SettingsTab_section">
                 <span className="mx_SettingsTab_subheading">{ _t("Account management") }</span>
                 <span className="mx_SettingsTab_subsectionText">
-                    { _t("Deactivating your account is a permanent action - be careful!") }
+                    { _t("Deactivating your account is a permanent action —— be careful!") }
```

Comment:
Shouldn't it be this?

```suggestion
                    { _t("Deactivating your account is a permanent action — be careful!") }
```

Reply:
I think em dash is — right ?

Reply:
Yes, but your PR has two em dashes right after each other i.e. this: ——